src/js_parser.zig

```diff
@@ -17425,8 +17460,13 @@ fn NewParser_(
 
                     if (comptime allow_macros) {
                         if (is_macro_ref and !p.options.features.is_macro_runtime) {
-                            const ref = e_.target.data.e_import_identifier.ref;
-                            const import_record_id = p.macro.refs.get(ref).?;
+                            const ref = switch (e_.target.data) {
+                                .e_import_identifier => |ident| ident.ref,
+                                .e_dot => |dot| dot.target.data.e_identifier.ref,
```

Comment:
I’m not really sure why `.e_dot` expressions have a target which is `.e_identifier` and not `.e_import_identifier` when the target is a namespace identifier, but I’m too afraid to change it.

Reply:
We remove these imports before they reach the final output